### ISSUE TITLE
Use availability_set  in HANA

### ISSUE THREAD

**Issue by risidoro** [SUSE/ha-sap-terraform-deployments#540]
Hi,
I would like to use an availability_set  that is already created instead of create a new one.

Thanks in advance.

**Comment by MalloZup** [comment]
hi @risidoro  can you provide more info about the `availabilty_set` acronynim? I didn't understand your question .. thx for your patience and feedback

**Comment by risidoro** [comment]
Hi Dario,
I'm working with a customer, and their HANA deployment(azure) they will use an existing availability set.
the resource is in the file `\azure\modules\hana_node\main.tf` and the resource is this one:
```
resource "azurerm_availability_set" "hana-availability-set" {
  name                        = "avset-hana"
  location                    = var.az_region
  resource_group_name         = var.resource_group_name
  managed                     = "true"
  platform_fault_domain_count = 2

  tags = {
    workspace = terraform.workspace
  }
}
```
I'm already done a work around adding a variable to the hana_module:
```
data "azurerm_availability_set" "hana-availability-set" {
  name                = var.hana-avset
  resource_group_name = var.resource_group_name
}
[ ... ]

resource "azurerm_virtual_machine" "hana" {
  [...]
  availability_set_id              = data.azurerm_availability_set.hana-availability-set.id
  [...]
```
Will be perfect is we can set the availability-set with the same approach like the vpc or the resource group

**Comment by yeoldegrove** [comment]
Azure Availability Sets cannot be configured in a granular way and you can have many of them in parallel.
Please elaborate on a concrete use case and reopen if needed.
